tests/datasource/data_connector/test_configured_asset_gcs_data_connector.py

```diff
@@ -25,6 +25,9 @@
         allow_module_level=True,
     )
 
+# module level markers
+pytestmark = [pytest.mark.external_cloud_client]
```

Comment:
Similar comment here. I feel `external_cloud_client` is probably too general because different ones may need different credentials.
If they just use mocks, make this `big` or `filesystem`.